integration/configs/browserCompatibilityTest/background_blur_test.config.json

```diff
@@ -11,7 +11,6 @@
       "testImpl": "BackgroundBlurTest.js",
       "payload": {
         "url": "http://localhost:8080/",
-        "port": 30002,
```

Comment:
Not really related but can you update the browser compatibility test to include backgroud replacement https://github.com/aws/amazon-chime-sdk-js/blob/main/integration/configs/integrationTest/background_blur_test.config.json#L19 (I missed it in  @davcolle previous PR)

Reply:
No problem, updated!